src/main/java/service/HorizontalServiceImpl.java

```diff
@@ -6,11 +6,11 @@
 
     private static final double dollarExchangeRate = 15;
     private BlindHorizontal blindHorizontal;
-    private PriceCatalog priceCatalog;
+    private PriceCatalogHorizontal priceCatalogHorizontal;
 
-    public HorizontalServiceImpl(BlindHorizontal blindHorizontal, PriceCatalog priceCatalog) {
+    public HorizontalServiceImpl(BlindHorizontal blindHorizontal, PriceCatalogHorizontal priceCatalogHorizontal) {
         this.blindHorizontal = blindHorizontal;
-        this.priceCatalog = priceCatalog;
+        this.priceCatalogHorizontal = priceCatalogHorizontal;
     }
 
     private double calculateAreaOfBlinds(int width, int height) {
```

Comment:
no need to write "Blinds" in the signature of the method. We already work with blinds

Reply:
Method calculateAreaOfBlinds renamed to calculateArea.